supervisor/addons/manager.py

```diff
@@ -67,6 +67,10 @@ def get(self, addon_slug: str, local_only: bool = False) -> AnyAddon | None:
             return self.store.get(addon_slug)
         return None
 
+    def get_local_only(self, addon_slug: str) -> Addon | None:
+        """Return an installed add-on from slug."""
```

Comment:
Should not be part of this PR, but maybe we should replace local with installed at one point, since we also have add-ons which are in a local "store".

Reply:
yea I actually wrote `get_installed_only` and then changed it back because it seemed like that was addressing a different problem.

Reply:
Yeah if we rename, we should also change local to installed in other places. Not part of this PR.